### ISSUE TITLE
the cowboy is feeding all the wheat to cows

### ISSUE THREAD

**Issue by linwetalion** [ldtteam/minecolonies#6021]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.3
- MineColonies Version: 0.13.401-BETA-universal
- Structurize Version:0.13.92-ALPHA-universal
- Related Mods and their Versions:forge 34.1.35

## Expected behavior
the cowboy should feed the wheat to cows only once to make them breed, after that he shall return to base

## Actual behavior
the cowboy feeds the cow all the wheat he has in stock, basically depleting all of the stock, repeatedly feeding the cows, that just have breed 1 second ago.

Another user had same issue (discord):
_ok my shepard and Cowherd always use all of the wheat i give them even tho the animal are not breedable so my wheat from my farm gets basically deleted(the 2 from the first breeding not included)_

he mentioned he has it in Minecolonies 13.394 structurize 0.13.94 forge 31.2.45

## Steps to reproduce the issue

1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
@someaddons

**Comment by Raycoms** [comment]
Is the cowboy feeding the children to grow up more quickly or is the cowboy giving more wheat to the adult cows?

**Comment by linwetalion** [comment]
> Is the cowboy feeding the children to grow up more quickly or is the cowboy giving more wheat to the adult cows?

he is giving wheat to the adult cows, that are unable to breed. re-checked it. 100% sure. goes from one to another and back, giving by 1 wheat at a time, until runs out of wheat, doesn't matter how much he has.

**Comment by linwetalion** [comment]
updated to minecolonies-0.13.416-ALPHA-universal.jar
Now the cowboy doesn't breed the cows.
When a cub grows up, the cowboy enters the farm, kills 1 cow randomly (hitting them all randomly until some of them falls dead) and after gathering loot returns to base and idles.
Restarting the worker doesn't help.

**Comment by Raycoms** [comment]
and after the killing, they will again reproduce the two, and then wait again, no?

**Comment by linwetalion** [comment]
no, after killing they all are just idling there, all three, 2 cows and a cowboy.
